userapp/src/components/user/UserForm.js

```diff
@@ -90,65 +109,76 @@ const UserForm = ({ initialUser, onSubmit}) => {
-                        label='Last Name'
-                        value={user.LastName}
-                        onChange={handleInputChange}
+                    error={error.LastName.length !== 0}
+                    name='LastName'
+                    label='Last Name'
+                    value={user.LastName}
+                    required={true}
+                    placeholder="Doe"
+                    helperText={error.LastName}
+                    onChange={handleInputChange}
                     />
-                    {/* {errors.LastName.length > 0 && <span className='error'>{errors.LastName}</span>} */}
                 </Grid>
                 <Grid item xs={6}>
                     <HelixTextField
-                        name='DateOfBirth'
-                        label='Date Of Birth'
-                        value={user.DateOfBirth}
-                        onChange={handleInputChange}
+                    name='DateOfBirth'
+                    label='Date Of Birth'
+                    type="date"
+                    value={user.DateOfBirth}
+                    InputLabelProps={{ shrink: true }}
+                    onChange={handleInputChange}
                     />
-                    {/* {errors.DateOfBirth.length > 0 && <span className='error'>{errors.DateOfBirth}</span>} */}
                 </Grid>
+                {error.DateOfBirth.length > 0 && <span className='error'>{error.DateOfBirth}</span>}
                 <Grid item xs={6}>
                     <HelixTextField
-                        name='Phone'
-                        label='Phone'
-                        value={user.Phone}
-                        onChange={handleInputChange}
+                    error={error.Phone.length > 0}
+                    name='Phone'
+                    label='Phone'
+                    value={user.Phone}
+                    helperText={error.Phone}
+                    onChange={handleInputChange}
+                    inputProps={{ maxLength: 10 }}
                     />
-                    {/* {errors.Phone.length > 0 && <span className='error'>{errors.Phone}</span>} */}
                 </Grid>
-                <Grid item xs={6}>
-                    <div>
-                        <Button 
-                        color="primary" 
-                        variant="contained" 
-                        type="submit" 
-                        size="large">
-                            Save
-                        </Button>
-                        <Button
-                        color="default"
-                        variant="contained"
-                        type="cancel"
-                        size="large"
-                        href="/">
-                            Cancel
-                        </Button>
-                    </div>
+                <Grid item xs></Grid>
+                <Grid item xs={6} className={userFormClasses.buttonStyle}>
+                    <Button 
```

Comment:
do we have Helix submit button/helix save button/helix cancel button

Reply:
Not yet